support-frontend/assets/pages/supporter-plus-landing/twoStepPages/checkoutScaffold.tsx

```diff
@@ -203,53 +220,69 @@ export function SupporterPlusCheckoutScaffold({
 				</FooterWithContents>
 			}
 		>
-			<CheckoutHeading
-				heading={!isPaymentPage && heading}
-				image={
-					!isPaymentPage &&
-					(showUsEoy2023Content ? (
-						<figure css={leftColImageUnitedStates}>
-							<GridImage
-								gridId="supporterPlusLandingUnitedStates"
-								srcSizes={[420]}
-								sizes="420px"
-								imgType="png"
-								altText=""
-							/>
-						</figure>
-					) : (
-						<figure css={leftColImage}>
-							<GridImage
-								gridId="supporterPlusLanding"
-								srcSizes={[500]}
-								sizes="500px"
-								imgType="png"
-								altText=""
-							/>
-						</figure>
-					))
-				}
-				withTopborder={isPaymentPage}
-			>
-				{!isPaymentPage &&
-					(showUsEoy2023Content ? (
-						<p css={subHeading}>
+			{isPaymentPage && (
```

Comment:
This is a nice refactor making changes easier between the steps.

Reply:
Thx